FrontEnd/src/components/basics/WizardForm/WizardForm.tsx

```diff
@@ -127,11 +144,13 @@ const WizardForm = <T extends {}>({
         { ...validations[curIndex] } as ValidationErrors<P>,
```

Comment:
feel like { success: true, error: undefined } can be extracted to be a template/const. Moreover, I believe the whole error logic can be written shorter and cleaner. I think we can just use the logic from lines 129 to 141 to handle both success and nonsuccess ones. Will result.error.formErrors still be defined if result success is true? If so, then we can definitely shorten them so that both cases can be handled in one go => when result success is true, fieldErrors[key as string] would always be false